src/tests.builds

```diff
@@ -5,6 +5,10 @@
   <ItemGroup>
     <TestProjectExclusions Include="$(MSBuildThisFileDirectory)*\tests\**\System.Security.AccessControl.Tests.csproj" />
     <TestProjectExclusions Include="$(MSBuildThisFileDirectory)*\tests\**\WebServer.csproj" Condition="'$(OS)' != 'Windows_NT'" />
+    <TestProjectExclusions Include="$(MSBuildThisFileDirectory)*\tests\**\System.Configuration.Tests.csproj" />
```

Comment:
We have to convert these to the `<Reference>` model since they are all new libraries that were added in master